class/defaults.yml

```diff
@@ -54,7 +54,7 @@ parameters:
               runbook_url: https://hub.syn.tools/appuio-cloud-reporting/runbooks/APPUiOCloudReportingMissingData.html
               summary: APPUiO Cloud Reporting is missing ERP link `target` data.
             expr: |
-              kube_job_failed{job="kube-state-metrics",namespace="${appuio_cloud_reporting:namespace}",job_name=~"check-missing-.*"} > 0
+              (kube_job_failed{job="kube-state-metrics",namespace="${appuio_cloud_reporting:namespace}",job_name=~"check-missing-.*"} * scalar((days_in_month() - day_of_month()) <bool 7)) > 0
```

Comment:
Should we make the last-n days of the month threshold configurable?

Reply:
No, I think this is good as is. We can change it when we have a need for it.